deltachat-ffi/deltachat.h

```diff
@@ -3951,7 +3974,7 @@ int64_t          dc_lot_get_timestamp     (const dc_lot_t* lot);
  *
  * @param data1 0
  * @param data2 (const char*) path name 
- *     Must not be free()'d or modified and is valid only until the callback returns.
+ *     Must not be released or modified and is valid only until the callback returns.
```

Comment:
i would prefer "unref'd" as it more clearly refering to all the dc_str_unref's but no biggie.